google/cloud/compute_v1/types/compute_small.py

```diff
@@ -0,0 +1,1033 @@
+# -*- coding: utf-8 -*-
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+import proto  # type: ignore
+
+
+__protobuf__ = proto.module(
+    package="google.cloud.compute.v1",
+    manifest={
+        "Address",
+        "AddressAggregatedList",
+        "AddressList",
+        "AddressesScopedList",
+        "AggregatedListAddressesRequest",
+        "Data",
+        "DeleteAddressRequest",
+        "Error",
+        "Errors",
+        "GetRegionOperationRequest",
+        "InsertAddressRequest",
+        "ListAddressesRequest",
+        "Operation",
+        "WaitRegionOperationRequest",
+        "Warning",
+        "Warnings",
+    },
+)
+
```

Comment:
This looks like it's coming from the "small" protos that are for testing.

https://github.com/googleapis/googleapis/blob/master/google/cloud/compute/v1/compute_small.proto

I think the owlbot match needs to be adjusted to exclude everything in https://github.com/googleapis/googleapis-gen/tree/master/google/cloud/compute/v1/compute-small-v1-py.

Reply:
Great catch! Fixed.